packages/hardhat-cannon/src/tasks/build.ts

```diff
@@ -26,18 +28,49 @@ task(TASK_BUILD, 'Assemble a defined chain and save it to to a state which can b
   )
   .addFlag('wipe', 'Start from scratch, dont use any cached artifacts')
   .addOptionalParam('preset', 'Specify the preset label the given settings should be applied', 'main')
-  .addOptionalVariadicPositionalParam('options', 'Key values of chain which should be built')
-  .setAction(async ({ noCompile, file, options, dryRun, port, preset, wipe }, hre) => {
+  .addOptionalVariadicPositionalParam('settings', 'Key values of chain which should be built')
+  .setAction(async ({ noCompile, file, settings, dryRun, port, preset, wipe }, hre) => {
     await installAnvil();
+
     if (!noCompile) {
       await hre.run(TASK_COMPILE);
+      console.log('');
     }
 
     const filepath = path.resolve(hre.config.paths.root, file);
 
-    // options can be passed through commandline, or environment
-    const mappedOptions: { [key: string]: string } = _.fromPairs((options || []).map((kv: string) => kv.split('=')));
     const def = loadCannonfile(hre, filepath);
+
+    if (!settings && !_.isEmpty(def.setting)) {
+      let displaySettings = Object.entries(def.setting).map((setting: Array<any>) => {
+        let settingRow: Array<any> = [
+          setting[0],
+          setting[1].defaultValue || dim('No default value'),
+          setting[1].description || dim('No description'),
+        ];
+        return settingRow;
+      });
+      console.log('This package can be built with custom settings.');
+      console.log(dim(`Example: npx hardhat cannon:build ${displaySettings[0][0]}="my ${displaySettings[0][0]}"`));
+      console.log('\nSETTINGS:');
```

Comment:
we need this kind of output on the cli as well

Reply:
probably even more important on the cli, actually

Reply:
Making a note and resolving comment